webboot/webboot.go

```diff
@@ -2,15 +2,55 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Get the time the machine has been up
+// Synopsis:
+//     webboot
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
```

Comment:
The comments show up twice. Remove the repeated comments

Reply:
Done